README.md

```diff
@@ -0,0 +1,3 @@
+# slides.cuban.tech
```

Comment:
`Diapositivas de los CubanTech meetups` , please change in English version as well

Reply:
Created README-es.md With the Spanish translation and done the translation in the original version.